console/src/assets/docs/QuickStartNewModelEN.md

```diff
@@ -0,0 +1,28 @@
+A Starwhale Model is a standard format for packaging machine learning models that contains the model file, inference codes, configuration files,
+
+**Step1: Install Starwhale Client by venv**
+
+```
+python3 -m venv ~/.cache/venv/starwhale
+source ~/.cache/venv/starwhale/bin/activate
+python3 -m pip install starwhale
+
+swcli --version
+
+sudo rm -rf /usr/local/bin/swcli
+sudo ln -s which swcli /usr/local/bin/
```

Comment:
This cmd is wrong, it will create wrong soft links:
- /usr/local/bin/swcli -> swcli
- /usr/local/bin/which -> which

We can merge these two lines to:

```bash
sudo ln -s "$(which swcli)" /usr/local/bin/
```

Reply:
done